components/FeedItemCard.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <v-hover>
+    <template v-slot="{ isHovering, props }">
+      <v-card v-bind="props">
+        <v-img :src="src()" aspect-ratio="1">
+          <template #placeholder>
+            <div class="d-flex align-center justify-center fill-height">
+              <v-container fluid class="fill-height">
+                <v-row class="ma-0">
+                  <v-col
+                    cols="12"
+                    class="pending font-weight-md-thin scale-text text-center"
+                  >
+                    {{ id }}
+                  </v-col>
+                  <v-col cols="12">
+                    <v-progress-linear
+                      indeterminate
+                      color="black"
```

Comment:
since we are using ad ark theme, the progress bar should be white.  there should be a way to swap out based on theme like a named color.  any of the colors defined in `plugins/vuetify.ts` in the themes i think you can put in this `color` attribute and it will use that color value.  e.g. `artby-gray` might work.  you can try adding your own